### PR TITLE
Adding the ability to translate tip text

### PR DESCRIPTION
**Pull request by Mingun**
Now the surrounding text in the template is not enclosed in the ``trans`` tag and therefore cannot be translated. This pull request fixing that.

### CODE REVIEW

**Comment by di** [comment]
Thanks! You'll need to rebuild the translations file as well, see https://warehouse.readthedocs.io/development/submitting-patches/#translations

**Comment by Mingun** [comment]
Done.

File: warehouse/templates/re-auth.html [hunk 1]
```diff
@@ -77,8 +77,8 @@ <h1 class="page-title">{% trans title=title %}Confirm password to continue{% end
               </span>
             </div>
             <span>
-              <b>Tip:</b> you are about to perform a <a href="{{ request.help_url(_anchor="sensitiveactions") }}">{% trans %}sensitive action{% endtrans %}</a>. If you are not on a personal computer, make sure to log out once you're done with your session.
-              We won't ask you to confirm your password again for the next hour.
+              {% trans href=request.help_url(_anchor="sensitiveactions") %}<b>Tip:</b> you are about to perform a <a href="{{ href }}">sensitive action</a>. If you are not on a personal computer, make sure to log out once you're done with your session.
+              We won't ask you to confirm your password again for the next hour.{% endtrans %}
```
Comment (di) [inline warehouse/templates/re-auth.html:81]:
```suggestion
              {% trans href=request.help_url(_anchor="sensitiveactions") %}
              <b>Tip:</b> you are about to perform a <a href="{{ href }}">sensitive action</a>.
              If you are not on a personal computer, make sure to log out once you're done with your session.
              We won't ask you to confirm your password again for the next hour.
              {% endtrans %}
```

**Comment by di** [comment]
Thanks!

**Pull request merged**